package.json

```diff
@@ -39,7 +39,7 @@
     "babel-preset-cozy-app": "1.9.1",
     "bundlesize": "0.18.0",
     "cozy-jobs-cli": "1.13.6",
-    "cozy-scripts": "1.13.2",
+    "cozy-scripts": "5.0.0",
```

Comment:
We should maybe change commit message (not to 4 but to 5) ?

Reply:
Fixed.